src/Model.elm

```diff
@@ -219,18 +210,31 @@ subscriptions model =
 
 fetchRecord : RecordId -> Cmd Msg
 fetchRecord recordId =
-    resourceConfig
-        |> Kinto.get recordId
+    recordConfig recordId
+        |> Kinto.get
+        |> Kinto.withDecoder (decodeData decodeRecord)
         |> Kinto.send FetchRecordResponse
 
 
-fetchRecords : Model -> Cmd Msg
-fetchRecords model =
-    Kinto.getRecordList
-        FetchRecordsResponse
-        model.kintoConfig
-        "default"
-        "test-items"
+fetchRecords : Cmd Msg
+fetchRecords =
+    recordListConfig
+        |> Kinto.get
+        |> Kinto.withDecoder (decodeData decodeRecords)
+        |> Kinto.send FetchRecordsResponse
+
+
+deleteRecord : RecordId -> Cmd Msg
+deleteRecord recordId =
+    recordConfig recordId
+        |> Kinto.delete
+        |> Kinto.withDecoder (decodeData decodeRecord)
+        |> Kinto.send DeleteRecordResponse
+
+
+decodeData : Decoder a -> Decoder a
+decodeData decoder =
+    field "data" decoder
```

Comment:
It's odd to me that this has to be defined in userland; I really feel like this should be handled by the lib itself, as a core, hidden implementation detail.